draft-ietf-scone-protocol.md

```diff
@@ -612,78 +612,81 @@ To reduce the risk of synchronization across multiple senders,
 which may cause network elements to miss updates,
 senders can include a small random delay.
 
+A network element MUST NOT alter datagrams to add SCONE packets
+or synthesize datagrams that contain SCONE packets.
+The latter will not be accepted and the former,
+even if they do not exceed the path MTU as a result,
+can be detected by applications and could be ignored.
+This document does not define a mechanism to support detection,
+but one might be added in future.
+
 
 ## Monitoring Flows {#monitoring}
 
-Network elements can monitor flows
-to determine which flows are from applications
-that respect throughput advice.
-This section outlines an exemplary algorithm
-for network elements.
+Sending throughput advice is optional for any network.
+A network that sends throughput advice might, also optionally,
+choose to monitor flows
+to determine whether applications are following advice.
+
+This section outlines a method
+that a network element could use
+to determine whether advice is being followed.
+Network deployments that choose to monitor
+are free to follow any monitoring regime that suits their needs.
 
 This monitoring algorithm is guidance only.
-Network elements are advised that monitoring
-any more strictly than the following likely invalidates their advice.
-Network elements could choose not to monitor in this way
-or use a looser monitoring approach.
-For instance, monitoring over a longer time window
+However, monitoring any more strictly than the following
+could mean that an application
+might be incorrectly classified as not following advice.
+A looser monitoring approach,
+such as monitoring over a longer time window
 than the monitoring period (67s)
-or using a higher rate than is signaled
-is possible.
-
-To operate this algorithm,
-the minimal state a network element maintains is:
-
-* the current throughput advice,
-* any state necessary to monitor throughput
-  (that is, throughput usage state, such as the state in {{sliding-window}}),
-* a timer used for rate increases, and
-* the time at which that throughput advice was last updated.
+or using a higher rate than is signaled,
+has no risk of incorrect classification.
 
+Network elements need to wait until advice is signaled.
 When advice is set or updated,
-the network element waits until
-it receives the next SCONE packet on affected flows.
-It then updates the throughput advice in that packet ({{apply}}).
+monitoring cannot start until the network element
+receives a SCONE packet
+and updates the throughput advice in that packet; see {{apply}}.
```

Comment:
If an element observes advice in a SCONE that is limited - either by the original sender or an upstream element - and does not update it, is it permitted to monitor?